### PR TITLE
Gxed Badger hack: use the same net/trace everywhere

### PR DESCRIPTION
**Pull request by hsanjuan**
net/trace cannot be imported on different versions from different modules.

Additionally, we also cannot export ExpVars with the same names from two different
packages (one being gxed-badger and the other being the go-modded one). Since those
two packages have different import paths, go considers them different and runs
init() twice, causing the issues.

### CODE REVIEW

**Comment by hsanjuan** [comment]
@Stebalien I have isolated this change here so that the migrations PR is cleaner.

File: go.mod [hunk 1]
```diff
@@ -1,3 +1,5 @@
 module github.com/ipfs/fs-repo-migrations
 
 go 1.13
+
+require golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa
```
Comment (Stebalien) [inline go.mod]: Let's still vendor this, just share it. The migrations should build even if all external deps go down.

**Comment by Stebalien** [comment]
If we use `go mod vendor`, we need to make sure to build with `go build -mod=vendor`. Although I guess this + go.sum meets the "works if everything else goes down" and "doesn't" change requirements.

**Comment by Stebalien** [comment]
That's why I avoided using `go mod vendor` for all of our other dependencies (e.g., gx, etc.). `go build` will still fail if the source goes down unless you specify `go build -mod=vendor`.

File: go.mod (hunk 1 above)
Comment (hsanjuan) [reply go.mod]: ok, I vendored this, and I made sure .travis.yml is building with vendored deps. If that works, it means we should be able to build migrations offline if needed.

**Comment by hsanjuan** [comment]
@Stebalien you commented while I was vendoring all the things properly. See now?

**Comment by hsanjuan** [comment]
So now travis will catch if things not vendored

**Pull request merged**